ruby_on_rails/app_initialisation.md

```diff
@@ -72,7 +71,22 @@ They are always idempotent (runnable multiple times).
   ```
 
   Also change `system!('bundle install')` to `system!('bundle install --jobs=3 --retry=3')`
-  and uncomment the installation of yarn dependencies with `system('bin/yarn')`
+
+#### If you are using a node-based JS bundler using `jsbundling-rails`
+
+* Add the following to `bin/setup` to allow intalling all dependencies in one go
+
+  ```ruby
+  system! 'yarn install'
+  ```
+
+#### If you use `importmap-rails` and there is no `bin/dev` file yet
+
+* Create one and make it executable. It will be used to start our project.
+
+  ```sh
+  echo "#\!/bin/sh\nset -e\n\nrails s" >> bin/dev && chmod +x bin/dev
+  ```
```

Comment:
When using forman (i.e. when using `jsbundling-rails`) the default generated script for running your app is `bin/dev`. Therefore I'd also change it to `bin/dev` for projects using importmaps. That way it is consistent between new projects.

Reply:
I'd keep consistency with all our previous project and go with a `bin/run` that either calls `bin/dev` or `bin/rails s`

Reply:
Yes that's a good idea. Let's keep it around.

Reply:
I think it's also convenient to keep `bin/run` since we have it also for non-rails projects. For example, I can run `bin/run` in the Renuo website project or a flutter app without even knowing the correct elixir/flutter command. 
Renaming it in all these projects seems bit weird just because the Rails standard changed :)

Reply:
+1